packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -103,14 +103,17 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
 
     const handleClose = (selection?: Option | null) => {
       if (!selection) {
-        if (inputValue !== selectedOption?.title) {
+        if (inputValue === "") {
```

Comment:
I guess you can just check if it's falsy value, empty string is always falsy

Reply:
I think I can, but is it any better? 🤔 I think I prefer to have it explicitly compared to the empty string. It seems more readable to me to understand that I want that to happen if the value of the field is empty? 💭